autoscheduler/frontend/src/package.json

```diff
@@ -37,7 +37,8 @@
     "typescript": "^3.6.4",
     "typescript-eslint": "0.0.1-alpha.0",
     "webpack": "^4.41.0",
-    "webpack-cli": "^3.3.9"
+    "webpack-cli": "^3.3.9",
+    "webpack-livereload-plugin": "^2.2.0"
```

Comment:
Since you updated `package.json`, shouldn't there be an according `package-lock.json` change with it?

Reply:
Apparently, it's [supposed to be included](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5), but for some reason, it didn't get added with `git commit -am`. I wonder if git stopped tracking changes because it got too long and we need to add it manually or something.